### PR TITLE
MC-9521: Added missing operations

### PR DESCRIPTION
**Pull request by fyrnaga**
Aligned descriptions to finish with a dot.
Added missing attributes in other operations

Added missing documentation for Openstack

### CODE REVIEW

**Review by patrickspensieri — APPROVED** [review]
✔️

File: source/includes/openstack/_volumes.md [hunk 1]
```diff
@@ -66,6 +70,9 @@ Attributes | &nbsp;
 `description`<br/>*string* | The volume's description
```
Comment (GuillaumeVialle) [inline source/includes/openstack/_volumes.md]: You forgot to add the dots here

File: source/includes/openstack/_volumes.md [hunk 2]
```diff
@@ -87,19 +94,14 @@ curl -X POST \
 
 <code>POST /services/<a href="#administration-service-connections">:service_code</a>/<a href="#administration-environments">:environment_name</a>/volumes</code>
 
-Create a volume
+Create a volume.
 
 Required attributes                | Description                         
 ---------------------------------- | -----------------------------------
 `name`<br/>*string*                | The volume name
```
Comment (GuillaumeVialle) [inline source/includes/openstack/_volumes.md]: Same here

Comment (fyrnaga) [reply source/includes/openstack/_volumes.md]: 18562f90c6d3e46d3ffc17d2814e50b4ebfa0ad0

File: source/includes/openstack/_volumes.md (hunk 1 above)
Comment (fyrnaga) [reply source/includes/openstack/_volumes.md]: 18562f90c6d3e46d3ffc17d2814e50b4ebfa0ad0

**Review by GuillaumeVialle — APPROVED** [review]
LGTM

**Pull request merged**